docs/whatsnew/6.0.rst

```diff
@@ -22,6 +22,29 @@ By the numbers:
 * X pull requests have been merged since v5.3
 * X distinct people have contributed code
 
+.. _whatsnew-6.0-geodetic-representation-geometry:
+
+Define Geodetic Representations via their geometric parameters
+==============================================================
+
+The `~astropy.coordinates.BaseGeodeticRepresentation` now exposes its geometric
+parameters via the attributes ``_equatorial_radius`` and ``_flattening``.
+The user may now define custom spheroidal models for the Earth or other Solar System
+bodies::
+
+    >>> from astropy.coordinates import BaseGeodeticRepresentation
+    >>> from astropy import units as u
+    >>> class IAU1976EarthGeodeticRepresentation(BaseGeodeticRepresentation):
+    ...     _equatorial_radius = 6378140 * u.m
+    ...     _flattening = 0.003352805 * u.dimensionless_unscaled
+    >>> representation = IAU1976EarthGeodeticRepresentation(lon=0.8*u.radian,
+    ...     lat=0.98*u.radian, height=367*u.m)
```

Comment:
This is really personal preference, but I think one might as well use `u.deg` for the example, since that is a more natural unit. Or, if one were to be fancy, something like `'32d10mW', '10d20mN'`